templates/default/rsyslog.conf.erb

```diff
@@ -76,6 +76,14 @@ $ActionFileDefaultTemplate <%= node["rsyslog"]["default_file_template"] %>
 $ActionFileDefaultTemplate RSYSLOG_TraditionalFileFormat
 <% end -%>
 
+#
```

Comment:
This should be within the if statement so we avoid adding a meaningless comment when users don't have templates set